bundles/com.espressif.idf.launch.serial.core/src/com/espressif/idf/launch/serial/internal/SerialFlashLaunchConfigDelegate.java

```diff
@@ -150,6 +164,34 @@ protected void launchInternal(ILaunchConfiguration configuration, String mode, I
 		DebugPlugin.newProcess(launch, p, String.join(" ", commands)); //$NON-NLS-1$
 	}
 
+	private void flashOverJtag(ILaunchConfiguration configuration, ILaunch launch) throws CoreException {
```

Comment:
Having the method modifier "protected" would help us to write a JUnit test case later.

Reply:
thanks for the review @kolipakakondal
changed to protected